db/models.py

```diff
@@ -36,17 +39,71 @@ def capacity(self) -> int:
         return self.rows * self.seats_in_row
 
     def __str__(self) -> str:
-        return self.name
+        return str(self)
 
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
     cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=CinemaHall, on_delete=models.CASCADE,
+        related_name="movie_sessions"
     )
     movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=Movie, on_delete=models.CASCADE,
+        related_name="movie_sessions"
     )
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
it is better to use AUTH_USER_MODEL instead of user, also what about related_name ?